lib/string-checker.js

```diff
@@ -89,7 +89,15 @@ StringChecker.prototype = {
         var parseError;
 
         try {
-            tree = this._esprima.parse(str, {loc: true, range: true, comment: true, tokens: true});
+            tree = this._esprima.parse(str, {
+                loc: true,
+                range: true,
+                comment: true,
+                tokens: true,
+                // TODO: sourceType is required to parse 'import' and 'export' statements,
```

Comment:
At a later date, we may consider revising this? I suppose I should just leave an issue, and not put the todo in code. Amending.